src/main/java/uk/gov/justice/probation/courtcasematcher/messaging/HearingExtractor.java

```diff
@@ -60,13 +62,17 @@ Hearing extractHearing(String payload, String messageId) {
         }
     }
 
-    private Hearing parseCPMessage(SnsMessageContainer snsMessageContainer) throws JsonProcessingException {
+    private List<Hearing> parseCPMessage(SnsMessageContainer snsMessageContainer) throws JsonProcessingException {
         final var cpHearingEvent = commonPlatformParser.parseMessage(snsMessageContainer.getMessage(), CPHearingEvent.class);
         final var hearing = cpHearingEvent.asDomain();
```

Comment:
could this be `final var hearings` since it's a list of hearings

Reply:
technically it can be